pkg/api/api.go

```diff
@@ -218,6 +218,12 @@ func (hs *HTTPServer) registerRoutes() {
 		r.Get("/user/auth-tokens/rotate", routing.Wrap(hs.RotateUserAuthTokenRedirect))
 	}
 
+	if hs.Features.IsEnabled(featuremgmt.FlagAuthenticationConfigUI) {
```

Comment:
Maybe we could move it into SAML package since it's enterpise only feature now?

Reply:
`hs.Index` is specifically hard to register in enterprise, it's not a small fix unfortunately

Reply:
@Jguer @mgyongyosi we have this, for example in report package

Reply:
didn't see it already embedded the http server without dep cycles, that's nice :+1:

Reply:
Circular dependencies are funny 😄